nemo/collections/nlp/models/ner_model.py

```diff
@@ -61,7 +61,7 @@ def __init__(
             :param activation: activation to usee between fully connected layers in the MLP
```

Comment:
these doc string seems to be in a wrong format. Please change, otherwise Spynx won't work on it

Reply:
Fixed.